modules/user_accounts/php/useraccounts.class.inc

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types=1);
+/**
+ * File contains the Users Accounts class.
+ *
+ * PHP Version 7
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\user_accounts;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+
+/**
+ * The modules class handles requests to module_manager/useraccounts/$username
+ * to retrieve or update information about a specific module.
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class UserAccounts extends \NDB_Page
```

Comment:
I think it's not a great idea to have two files `user_accounts.class.inc` and `useraccounts.class.inc`. Can you change this to something more descriptive? I feel like this can only lead to confusion

Reply:
I would call it `UserAccountsEndpoint`. But in another PR @driusan pointed out that the `Endpoint` part seems unnecessary?

What are your thought @driusan?

Reply:
After some reflection on what Dave said in the comments, I don't think this class needs for this PR because this class is more for creating real Loris Endpoints.